src/civetweb.c

```diff
@@ -59,6 +59,14 @@
 #pragma warning(disable : 4127)
 /* non-constant aggregate initializer: issued due to missing C99 support */
 #pragma warning(disable : 4204)
+/* padding added after data member */
+#pragma warning (disable : 4820)
+/* not defined as a preprocessor macro, replacing with '0' for '#if/#elif' */
+#pragma warning (disable : 4668)
+/* no function prototype given: converting '()' to '(void)' */
```

Comment:
I think I actually added that one because it was complaining about the MSVC stdlib headers. We can pull it out though and check.